app-sso/getting-started/appsso-overview.hbs.md

```diff
@@ -78,3 +80,207 @@ applications may verify their validity using the AuthServer's public keys.
+
+### Verify that your AuthServer is running
+
+You can now see the service offering with:
+
+```shell
+tanzu services classes list
+```
+
+You should see
+
+```shell
+NAME                          DESCRIPTION
+my-login  Login by AppSSO - user:password - UNSAFE FOR PRODUCTION!
+```
+
+>**Note:** As you can see this Login offering is *not* safe for production as it hard codes a user and password, but
+it is the quickest way to get started.
+
+You can wait for the `ClusterUnsafeTestLogin` to become ready with:
+
+```shell
+kubectl wait --for=condition=Ready clusterUnsafeTestLogin my-login
+```
+
+Alternatively, you can inspect your `ClusterUnsafeTestLogin` like any other resource:
+
+```shell
+kubectl get clusterunsafetestlogin.sso.apps.tanzu.vmware.com --all-namespaces
+```
+
+and you should see:
+
+```shell
+NAME                          STATUS
+my-login  Ready
+```
+
+If you want to see your `AuthServer`'s issuer URI, look at:
+
+```shell
+kubectl get authserver -A
+```
+
+## Claim Credentials
+
+Now that you have an `AppSSO` service offering, the next step is to create a `ClassClaim`.
+
+This creates consumable credentials for your workload, and will allow your workload to connect to the Login service
+offering using those credentials.
+
+![Diagram of the Connection between your Workload, the ClassClaim, and AppSSO.](../../images/app-sso/appsso-flow.png)
+
+Choose your Login offering out of the options available at
+
+```bash
+tanzu services classes list
+```
+
+If there aren't any available, [you can set one up.](#provision-an-authserver)
+
+```bash
+tanzu services class-claims create my-workload \
+  --class my-login \
+  --parameter workloadRef.name=appsso-starter-java \
+  --parameter redirectPaths='["/login/oauth2/code/appsso-starter-java"]'
+```
+
+>**Note:** The redirect path you need to provide is referring to the login redirect within your application. In our case,
+we will be deploying a minimal Spring application, so we will use the Spring Security path.
+
+Check the status of your ClassClaim with
+
+```bash
+tanzu services class-claims list
+```
+
+# Deploy an application with Application Single Sign-On
+
+This topic tells you how to deploy a minimal Kubernetes application that is protected
+by Application Single Sign-On (commonly called AppSSO) by using the credentials
+that [tanzu services class-claims](#claim-credentials) creates.
+
+![Diagram of AppSSO's components and how they interact with End-Users and Client applications](../../images/app-sso/appsso-concepts.png)
+
+[//]: # (^ diagram is produced from https://miro.com/app/board/uXjVMUY5O0I=/)
+
+For more information about how a Client application uses an AuthServer to
+authenticate an end user, see the [Overview of AppSSO](./appsso-overview.md).
+
+## Prerequisites
+
+- You must complete the steps described in [Get started with Application Single Sign-On](./appsso-overview.hbs.md).
+  If not, see [claim credentials](#claim-credentials).
+
+## Deploy a minimal application
```

Comment:
[issue]: same anchor issue as above -- missing `a` tag.

[issue]: The table of contents text for this anchor is different from the text here. "Deploy your workload" vs here "Deploy a minimal application